Android documentation

kevinliao2016 opened the pull request:
@Jayroo5245  please review the comments I put in

Jayroo5245 commented:
+0 - Please address comments, merge to resolve conflicts and upload a new patch. Thanks @kevinliao2016

Jayroo5245 commented:
@kevinliao2016  This PR still says "This branch has conflicts that must be resolved". Please work with @ElViajero to see how to resolve conflicts if needed.

Jayroo5245 commented:
Also, please make sure to assign it to me when you are ready (you may not have been, i just decided to swoop in and take a look ;-) )

kevinliao2016 commented:
@Jayroo5245 
the previous commit:
- resolved the conflicts
- included email fix and set fab to "gone"
- renamed getUserData()
-removed unnecessary comments

:+1:

Jayroo5245 commented:
Just curious did you merge from development into your local branch already @kevinliao2016 ?

+2 - You can merge to development.

kevinliao2016 commented:
On this site, it kept saying "This branch has conflicts that must be resolved". I was on the androidDocumentation Branch, the bash terminal showed the branch was clean. I merged local master into this branch and saw bunch of conflicts in some places I never touched. so I deleted the heads and did a push again. Then it showed "This branch has no conflicts with the base branch"
